multires-lyrics-search/README.md

```diff
@@ -1,109 +1,176 @@
+
+
+
+# Flow diagram
+This diagram provides a visual representation of the two Flows in this example. Showing which executors are used in which order.
+
+
+![Untitled (10)](https://user-images.githubusercontent.com/59612379/116891336-dd86bc80-ac2e-11eb-9d77-f301846688e3.jpg)
+
+
+# Next steps, building your own app
+
+Did you like this example and are you interested in building your own? For a detailed tuturial on how to build your Jina app check out [How to Build Your First Jina App](https://docs.jina.ai/chapters/my_first_jina_app/#how-to-build-your-first-jina-app) guide in our documentation. 
+
+For more information on chunking, check out our [Understand Jina Recursive Document Representation](https://docs.jina.ai/chapters/traversal/#understand-jina-recursive-document-representation).
+
+If you have any issues following this guide, you can always get support from our [Slack community](https://join.slack.com/t/jina-ai/shared_invite/zt-dkl7x8p0-rVCv~3Fdc3~Dpwx7T7XG8w) .
+
+# 🐳 Deploy the prebuild application using Docker
+To make it easier for you, we have built and published the [Docker image](https://hub.docker.com/r/jinahub/app.example.multireslyricssearch) with 10000 indexed songs (more than the toy example, but just a small part of the huge dataset).
+
+## Pre-requirements:
+1. You have Docker installed and working. 
+
+2. You have at least 8GB of free space on your hard drive. 
+
+
+## Step 1. Pull and run the image
+Running the following command will pull the Docker image and run it.
 
 ```bash
 docker run -p 65481:65481 jinahub/app.example.multireslyricssearch:0.0.2-0.9.20
 ```
 
 
-## 🏃 Run the Flows
-Now that we've got the code to load our data, we're going to dive into writing our app and running our Flows!
-### Index Flow
-To run the index you type:
-```bash
-python app.py -t index
-```
-First up we need to build up an index and then search through this index when we use the query Flow later.
-Then we have ready all the indexes!
-### Query Flow
-Now for the query time, run:
-```bash
-python app.py -t query
-```
+# Community
 
-### With REST API
+- [Slack channel](https://join.slack.com/t/jina-ai/shared_invite/zt-dkl7x8p0-rVCv~3Fdc3~Dpwx7T7XG8w) - a communication platform for developers to discuss Jina
+- [Community newsletter](mailto:newsletter+subscribe@jina.ai) - subscribe to the latest update, release and event news of Jina
+- [LinkedIn](https://www.linkedin.com/company/jinaai/) - get to know Jina AI as a company and find job opportunities
+- [![Twitter Follow](https://img.shields.io/twitter/follow/JinaAI_?label=Follow%20%40JinaAI_&style=social)](https://twitter.com/JinaAI_) - follow us and interact with us using hashtag `#JinaSearch`  
+- [Company](https://jina.ai) - know more about our company, we are fully committed to open-source!
+
+# License
+
+Copyright (c) 2021 Jina AI Limited. All rights reserved.
+
+Jina is licensed under the Apache License, Version 2.0. See LICENSE for the full license text.
```

Comment:
Link to license

Reply:
What is the link?

Reply:
Do we have LICENSE file in the repo root? If not, should add it, then link there

Reply:
Done!